packages/manager/apps/web/client/app/hosting/database/copy/hosting-database-copy.html

```diff
@@ -0,0 +1,106 @@
+<div data-ng-controller="HostingDatabaseCopyCtrl as $ctrl">
+    <div
+        data-wizard
+        data-wizard-bread-crumb
+        data-wizard-on-cancel="resetAction"
+        data-wizard-on-finish="copyDatabase"
```

Comment:
**Question** : not possible to use `$ctrl.resetAction` and `$ctrl.copyDatabase` ?

Reply:
Validated with David: wizard component does not allow that since it's start functions in its own context.